content/blog/2023/06 /02

```diff
@@ -0,0 +1,17 @@
+title: "The journey of Letica and development assessments - Research Call with Jonathan Reams"
+created: 2023-06-02
+categories: 
+  - "research-call"
+authors: 
+  - Rufuspollock
+
+>This call forms part of our monthly Research Call events. To get involved with this and our community calls please join the Life Itself [Whatsapp Chat](https://chat.whatsapp.com/JNJCTZugNQn)
+
+**When:** 15:00CET, Friday June 9th, 2023
+**Who:** Jonathan Reams is an Associate Professor at the Norwegian University of Science and Technology and has an insatiable curiosity about the essence of human nature and how to cultivate this essence in the service of leadership. He is also one of the leading experts on Letica in Europe. Jonathan will discuss the journey so far for Lectica over the past 20 years, along with his experience in applying Lectica, and the future vision of the application of developmental assessments, before opening out the call into a wider discussion.
```

Comment:
You don't need who here since it would be same as what (if we have who it is usually about who can come). So if you had it would be:

```
**Who**: open to all
```

Reply:
I would have used this blurb for the lead-in or have it in more info below (you currently have two versions of the blurb and would just have one).